### ISSUE TITLE
Tales of Vesperia: Definitive Edition

### ISSUE THREAD

**Issue by GoffyDude** [Ryujinx/Ryujinx-Games-List#1020]
#### Current on `master` : 1.0.4390

- Game loads to menus with corrupt graphics (garbled text).
- Plays initial cutscenes well, apart from menu overlays still having garbled text. Later on, interactive cutscenes have garbled text as well as distorted audio.
- Goes in-game, but the garbled text bug makes the game unplayable.

#### Screenshots :

![Ryujinx_0uL0sYoeO0](https://user-images.githubusercontent.com/62343878/80812529-6636cc80-8b85-11ea-8752-2111e5d893e5.png)
![Ryujinx_afM1RlWXx9](https://user-images.githubusercontent.com/62343878/80812541-6afb8080-8b85-11ea-9da5-bde6dda00eea.png)
![Ryujinx_DrK4N59nwv](https://user-images.githubusercontent.com/62343878/80812570-73ec5200-8b85-11ea-9862-e0bd7d1c564c.png)
![Ryujinx_Hqqn8eNbPC](https://user-images.githubusercontent.com/62343878/80812575-751d7f00-8b85-11ea-9be7-3adb05dcac44.png)
![Ryujinx_imtStIaYq0](https://user-images.githubusercontent.com/62343878/80812598-7ea6e700-8b85-11ea-8d5c-dc4fdf6c21ce.png)
![Ryujinx_KrdZ05XBEo](https://user-images.githubusercontent.com/62343878/80812608-82d30480-8b85-11ea-9792-d5551ab5a9fa.png)
![Ryujinx_RrtiGr2jH7](https://user-images.githubusercontent.com/62343878/80812616-88c8e580-8b85-11ea-90bf-aca8bbc6324e.png)
![Ryujinx_uPYEh5YYZp](https://user-images.githubusercontent.com/62343878/80812618-89617c00-8b85-11ea-8b65-5339ef623b0b.png)
![Ryujinx_YM58yvYZYi](https://user-images.githubusercontent.com/62343878/80812630-90888a00-8b85-11ea-9b2a-8630f359b910.png)

#### Log file :

[ToVDE.zip](https://github.com/Ryujinx/Ryujinx-Games-List/files/4564697/ToVDE.zip)

**Comment by EmulationFanatic** [comment]
Same status on 1.0.6407

**Comment by EmulationFanatic** [comment]
Updated

**Comment by Zyute1991** [comment]
I have just tried it and with version 1.0.2 it works perfectly at native resolution. Anything above native drops the fps fast. At 2x its around 50fps and anything higher i would say is unplayable. In my opinion it should be moved from nothing to ingame due to it running at 60fps native with no game breaking glitches.

My specs are
I9-10850k
RTX 3080
32gb ddr4

**Comment by deusnovus** [comment]
I encountered two problems with this game I'd like to report:

1. The game won't launch after the first boot, unless you purge the shader cache and start over.
2. The in-game audio is very grainy, while the intro cutscene audio is fine. (it worked properly only once for some reason and haven't been able to replicate it ever since) [Here is an example of the grainy audio.](https://www.dropbox.com/scl/fi/iqt18ayopbazo5zxox2n6/talesofvesperia.mp4?rlkey=muxscgv4m1bh5knppvmfhvjf0&dl=0)

I tried all of the above with all available versions (v1.0 and v1.0.2), with all audio/video drivers (OpenGL, Vulkan iGPU and NVIDIA) and it produces the same problem.

**Log after the shader crash:** 
[Ryujinx_1.1.999_2023-08-22_20-34-46.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/12411285/Ryujinx_1.1.999_2023-08-22_20-34-46.log)

**Log during the grainy audio:**
[Ryujinx_1.1.999_2023-08-22_20-34-46.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/12411335/Ryujinx_1.1.999_2023-08-22_20-34-46.log)

**System info:**

Dell Inspiron 16 Plus 7610
CPU: 11th Gen Intel® Core™ i7-11800H
RAM: 32GB
GPU1: NVIDIA GeForce RTX™ 3050 Ti Laptop GPU
GPU2: Intel® UHD Graphics (TGL GT1)
OS: Fedora 38 / GNOME 44.3 / Wayland
Ryujinx version: 1.1.999

**Comment by GamerzHell9137** [comment]
> I encountered two problems with this game I'd like to report:
> 
>     1. The game won't launch after the first boot, unless you purge the shader cache and start over.
> 
>     2. The in-game audio is very grainy, while the intro cutscene audio is fine. (it worked properly only once for some reason and haven't been able to replicate it ever since) [Here is an example of the grainy audio.](https://www.dropbox.com/scl/fi/iqt18ayopbazo5zxox2n6/talesofvesperia.mp4?rlkey=muxscgv4m1bh5knppvmfhvjf0&dl=0)
> 
> 
> I tried all of the above with all available versions (v1.0 and v1.0.2), with all audio/video drivers (OpenGL, Vulkan iGPU and NVIDIA) and it produces the same problem.
> 
> **Log after the shader crash:** [Ryujinx_1.1.999_2023-08-22_20-34-46.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/12411285/Ryujinx_1.1.999_2023-08-22_20-34-46.log)
> 
> **Log during the grainy audio:** [Ryujinx_1.1.999_2023-08-22_20-34-46.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/12411335/Ryujinx_1.1.999_2023-08-22_20-34-46.log)
> 
> **System info:**
> 
> Dell Inspiron 16 Plus 7610 CPU: 11th Gen Intel® Core™ i7-11800H RAM: 32GB GPU1: NVIDIA GeForce RTX™ 3050 Ti Laptop GPU GPU2: Intel® UHD Graphics (TGL GT1) OS: Fedora 38 / GNOME 44.3 / Wayland Ryujinx version: 1.1.999

Ryujinx has more than one audio backend. Have you tried all of them?

**Comment by GamerzHell9137** [comment]
Also the log of the shader crash is empty. If its not showing in logs then screenshot the command window while its crashing.

**Comment by deusnovus** [comment]
> Ryujinx has more than one audio backend. Have you tried all of them?

Yes, both OpenAL and SDL2 have the same grainy audio. SoundIO crashes the game. I managed to get proper in-game only once with SDL2 in random, haven't been able to reproduce it ever since.

> then screenshot the command window while its crashing.

How do I view the console window in Linux?

**Comment by raptorthesoul** [comment]
my game doesnt boot after the first time a swell. have there been any changes?

**Comment by Snipy44** [comment]
same problem here, vesperia crash my game
